src/svgcontext.ts

```diff
@@ -148,7 +155,8 @@ export class SVGContext extends RenderContext {
     };
 
     this.state = {
-      scale: { x: 1, y: 1 },
+      scaleX: 1,
+      scaleY: 1,
```

Comment:
I have refactored those two in order to avoid a more complex definition of `Attributes`